packages/indexer/.env.example

```
@@ -7,6 +7,10 @@ PORT=3000
 # Chain id the indexer is running on
 CHAIN_ID=1
 
+# Custom: Imtbl Orderbook
+IMTBL_ORDERBOOK_MAKER_FEE_BPS=200 # 100 == 1%
```

The comment created issues before on deployments, where it would include the comment in the value.

```suggestion
IMTBL_ORDERBOOK_MAKER_FEE_BPS=200
```

---

changed in next commit